sdk/keyvault/microsoft-azure-keyvault-complete/pom.xml

```diff
@@ -41,22 +41,27 @@ the MIT License. See License.txt in the project root for license information. --
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-keyvault</artifactId>
+      <version>1.2.2</version>
```

Comment:
we can extract this into a property.

Reply:
applies to each pom

Reply:
For consistency with the way we are specifying internal dependencies within core (just as one example) I think we should leave this as explicit. I think moving to explicit version references will also help when we shift to automated tooling to synchronize versions (because that tooling won't then need to go and interpret properties.